### ISSUE TITLE
Not tracking Funnel Bats - Bostaunieux Oubliette, Page 4

### ISSUE THREAD

**Issue by nightpanda2810** [onimitch/ffxi-trainmon#2]
As I kill funnel bats (the UI for the addon says "Funnel Bat") the counter does not go up. Works for the Werebats on the same page. Counter corrects the funnel bats when checking reviewing at the grounds tome however.

**Comment by onimitch** [comment]
Any chance you could share a screenshot of what the Trainmon UI has and what your chat log shows when you kill a Funnel Bat? Also let me know what Zone this is in. Would help me fix it a lot quicker, thanks!

**Comment by nightpanda2810** [comment]
So I keep running into this bug in multiple zones, and it seems to keep fixing itself after a few completions. I don't even know how to duplicate it anymore LOL. I suspect another plugin I have may be interfering.

**Comment by onimitch** [comment]
Yeah it's quite possible. Unfortunately trainmon in it's current state relies entirely on parsing the chat log for kill messages, so if for some reason they don't arrive or another addon reformats them, it can mess with trainmon.

At some point I'd like to rework the kill detection to get the packet data and not rely on parsing the chat logs, but not sure if/when I'll get the time to do that.
